actions.go

```diff
@@ -0,0 +1,55 @@
+package web
+
+import (
+	"context"
+	"embed"
+	"io/fs"
+
+	"github.com/launchrctl/launchr/pkg/action"
+	"github.com/launchrctl/launchr/pkg/cli"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	webTestAct = "web.test"
```

Comment:
Let's look for action name dynamically to make this file `actions.go` generic and copy/pastable to other plugins commands

Reply:
it's not an issue to dynamically create actions, but issue in action callback developer wants to apply to actions. 

this code still needs to be created by dev and attached to so appropriate idss

```
var testFunc action.ServiceCallbackFunc = func(ctx context.Context, input action.Input) error {
		cli.Println("Executing example function...")
                // todo

		return nil
	}
```